examples/postgres/appengine-standard-java8/README.md

```diff
@@ -0,0 +1,54 @@
+# Postgre SQL sample for Google App Engine for Java 8
+This sample demonstrates how to use [PostgreSql](https://cloud.google.com/sql/) on Google App
+Engine standard Java 8
+
+## Setup
+
+* If you haven't already, Download and initialize the [Cloud SDK](https://cloud.google.com/sdk/)
```

Comment:
Download -> download

Don't capitalize non-proper names in the middle of a sentence.